src/style.css

```
@@ -0,0 +1,79 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  --block_height: 300px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: var(--block_height);
+  min-width: 300px;
+  line-height: 300px;
+  text-align: center;
+  flex-basis: 33.33%;
+  flex-grow: 1;
+}
+
+.block__1 {
+  order: 1;
+  background-color: rgb(255 0 0);
```

use commas to separate values, it'll improve readability
```suggestion
  background-color: rgb(255, 0, 0);
```